packages/kas/src/nodes.ts

```diff
@@ -162,65 +212,54 @@ class Expr {
 
     // returns the name of this expression's constructor as a string
     // only used for testing and debugging (the ugly regex is for IE8)
-    name() {
-        if (this.func.name) {
-            return this.func.name;
-        } else {
-            return this.func.toString().match(/^function\s*([^\s(]+)/)[1];
-        }
+    name(): string {
+        return this.func.name;
     }
 
     // returns a string representing current node structure
-    repr() {
+    repr(): string {
         return (
             this.name() +
             "(" +
             _.map(this.args(), function (arg) {
-                return _.isString(arg) ? arg : arg.repr();
+                return _.isString(arg) || _.isNumber(arg) ? arg : arg.repr();
             }).join(",") +
             ")"
         );
     }
 
     // removes all negative signs
-    strip() {
+    strip(): Expr {
         return this.recurse("strip");
     }
 
     // canonically reorders all commutative elements
-    normalize() {
+    normalize(): Expr {
         return this.recurse("normalize");
     }
 
     // expands the expression
-    expand() {
+    expand(options?): Expr {
```

Comment:
Since `options` isn't being used, can it be omitted?

Reply:
Yes it can.  I'll make this change.